src/status_im/node/core.cljs

```diff
@@ -74,13 +74,30 @@
       (if utils.platform/desktop? ""
           config/log-level-status-go)))
 
+(def default-fleets (slurp "resources/config/fleets.json"))
+
+(defn fleets [{:keys [custom-fleets]}]
+  (as-> [(default-fleets)] $
+    (mapv #(:fleets (types/json->clj %)) $)
+    (conj $ custom-fleets)
+    (reduce merge $)))
+
+(defn current-fleet-key [db]
+  (keyword (get-in db [:multiaccount :settings :fleet]
+                   config/fleet)))
+
+(defn get-current-fleet
+  [db]
+  (get (fleets db)
+       (current-fleet-key db)))
+
```

Comment:
Why do we need to move this code form `fleet.core` to `node.core`?

Reply:
`fleet.core` is mostly about saving a new fleet config and to do that you need to call `node.core`. So if `get-current-fleet` is in `fleet.core` as well you get a circular dependency. I suppose we could make a fleet.db ns otherwsie. Right now it's similar to `log-level.core` and `get-log-level` is in `node.core` too.

Reply:
I would go for that `fleet.something` ns, then we actually don't need to make changes in other namespaces except `fleet.core` and `node.core`.

Reply:
ok but shouldn't I change log-level namespace as well for consistency then? Also it will still change other namespaces, but just the ns imported.

Reply:
We def can do it without changes in other namespaces, the call can go through fleet.core, not a big deal though. 

Regarding `log-level`, it's not objective of this PR so I wouldn't do this here. Actually if everyone is fine with putting that fleet related code into `node.core` no problem. But it's a bit puzzling then what are `fleet.core` and `node.core` responsible for.